07_train_final_classifiers/collect_full_models.ipynb

```diff
@@ -0,0 +1,706 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.                .reindex(all_feats)
</pre><p>If I'm following this correctly, this assumes that rows in <code>results_file</code> match exactly the order of <code>all_feats</code> that you assembled in the previous cell, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/mpmp/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I don't think so - <code>reindex</code> should re-order the existing index if it isn't in the same order. (Some of the examples in <a href="Some of the examples in [the docs here](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.reindex.html" target="_blank">the docs here</a> show this)</p><p> </p><p>It does assume that all of <code>all_feats</code> are found in the index of <code>results_file</code> (if not it will add NaN rows - I'll add a test for that).   </p>